src/Version.php

```diff
@@ -32,11 +32,56 @@ public static function isStable()
     }
 
     /**
-     * Compares a version to Bolt's version.
+     * Compares a semantic version against Bolt's version, given a specified
+     * comparison operator.
      *
-     * Note: Be sure to include the `.z` number in the version given, as omitting it can give inconsistent results.
-     * For example, if the current version is `3.3.0`, `compare('3.3', '>=')` returns false. In reality you want
-     * the check to return true for this case, and `compare('3.3.0', '>=')` _does_ return true.
+     * @see http://semver.org/ For an exmplaination on semantic versioning.
+     *
+     * Note: Be sure to include the `.z` number in the version given, as
+     * omitting it can give inconsistent results. See below at the results in
+     * Tables 1 & Table 2 for more detail.
+     *
+     * Note: Depending on the stability of the VERSION constant, the output of
+     * this may give, initially, unexpected results.
```

Comment:
Well, more what it is trying to say is, if there is a non-stable part to the version … blah